### PR TITLE
karate file added by mabrur

### PR DESCRIPTION
**Pull request by Horizon-dev4**

### CODE REVIEW

**Review by horizondev1 — CHANGES_REQUESTED** [review]
please change the varibale name

File: car.js [hunk 1]
```diff
@@ -0,0 +1 @@
+console.log("lalmborghini is the fastest car");
```
Comment (horizondev1) [inline car.js:1]: please change car varibale to superCar

Comment (Horizon-dev4) [reply car.js:1]: you ask for a lot of changes. I wont do that now. Don't have time....

Comment (horizondev1) [reply car.js:1]: you should complete full project in today

Comment (Horizon-dev4) [reply car.js:1]: ok, I will fix it.

**Review by horizondev1 — APPROVED** [review]
good good

**Pull request merged**